microdnf/commands/groupinfo/groupinfo.cpp

```diff
@@ -160,9 +160,7 @@ void GroupinfoCommand::run() {
             installed_groupids.insert(group.get_groupid());
         }
         for (auto group: query_available.list()) {
-            if (installed_groupids.find(group.get_groupid()) == installed_groupids.end()) {
-                group_list.emplace(group);
-            }
+            group_list.emplace(group);
```

Comment:
Does this really list installed groups twice then? I don't think that's a good UI...

Reply:
It prints 2 **different** groups.
It is good to know that a group has been installed **and** that there's a different group available.
Maybe we should highlight "Installed: yes/no",  maybe move it up in the listing so it's more obvious.
It's also clear we need to reformat the list of packages, but that's a matter of creating another PR.

$ microdnf groupinfo --hidden core
(please note I manually copied the @core group from test data to /var/lib/dnf/system-state/comps-installed.xml.zst)
```
Group id             : core
Name                 : Core
Description          : Smallest possible installation
Order                : 1
Langonly             : it
Uservisible          : False
Installed            : True
Repositories         : @System
Mandatory packages   : 
  bash               : 
  glibc              : 
Default packages     : 
  dnf                : 
Optional packages    : 
  dnf-plugins-core   : 
Conditional packages : 
  conditional        : 

Group id                  : core
Name                      : Core
Description               : Smallest possible installation
Order                     : 
Langonly                  : 
Uservisible               : False
Installed                 : False
Repositories              : rawhide
Mandatory packages        : 
  audit                   : 
  basesystem              : 
  bash                    : 
  coreutils               : 
  curl                    : 
  dhcp-client             : 
  dnf                     : 
  e2fsprogs               : 
  filesystem              : 
  glibc                   : 
  grubby                  : 
  hostname                : 
  iproute                 : 
  iputils                 : 
  kbd                     : 
  less                    : 
  man-db                  : 
  ncurses                 : 
  openssh-clients         : 
  openssh-server          : 
  parted                  : 
  passwd                  : 
  policycoreutils         : 
  procps-ng               : 
  rootfiles               : 
  rpm                     : 
  selinux-policy-targeted : 
  setup                   : 
  shadow-utils            : 
  sssd-common             : 
  sssd-kcm                : 
  sudo                    : 
  systemd                 : 
  util-linux              : 
  vim-minimal             : 
  yum                     : 
Default packages          : 
  NetworkManager          : 
  dnf-plugins-core        : 
  dracut-config-rescue    : 
  fedora-repos-modular    : 
  firewalld               : 
  plymouth                : 
  systemd-oomd-defaults   : 
  systemd-resolved        : 
  zram-generator-defaults : 
Optional packages         : 
  dracut-config-generic   : 
  initial-setup           : 
  initscripts             : 
Conditional packages      : 
```